docs/tutorials/google/spin_echoes.ipynb

```diff
@@ -0,0 +1,1362 @@
+{
```

Comment:
<p>Again, let's hint that Cirq supports decomposing to many different target gatesets via analytical decomposition functions present in https://github.com/quantumlib/Cirq/tree/master/cirq-core/cirq/optimizers. As an example, users can use <code>two_qubit_matrix_to_sqrt_iswap_operations</code>  to compile to <code>sqrt(iSwap)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4373/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Reworded and added href to <code>cirq.optimizers</code>  to hint at the larger support.</p>